cmd/update/execution.go

```diff
@@ -0,0 +1,73 @@
+package update
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/flyteorg/flytectl/clierrors"
+	"github.com/flyteorg/flytectl/cmd/config"
+	"github.com/flyteorg/flytectl/cmd/config/subcommand/execution"
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
+	"github.com/flyteorg/flytestdlib/logger"
+)
+
+const (
+	updateExecutionShort = "Update execution status"
+	updateExecutionLong  = `
+Activating execution unhides it from cli and UI:
+::
+
+ flytectl update execution -p flytectldemo -d development  oeh94k9r2r --activate
+
+Archiving execution hides it from cli and UI:
+::
+
+ flytectl update execution -p flytectldemo -d development  oeh94k9r2r --archive
+
+
+Usage
+`
+)
+
+func updateExecutionFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
+	project := config.GetConfig().Project
+	domain := config.GetConfig().Domain
+	if len(args) != 1 {
+		return fmt.Errorf(clierrors.ErrExecutionNotPassed)
+	}
+	executionName := args[0]
+	activateLP := execution.UConfig.Activate
+	archiveLP := execution.UConfig.Archive
+	if activateLP == archiveLP && archiveLP {
```

Comment:
nit: maybe `if activateLP && archiveLP {` for readability?

Reply:
Done